app/marketing/mails.py

```diff
@@ -300,18 +301,24 @@ def new_supporter(grant, subscription):
         translation.activate(cur_language)
 
 
-def thank_you_for_supporting(grant, subscription):
-    if subscription and subscription.negative:
-        return
+def thank_you_for_supporting(grants, contributor):
```

Comment:
Ah this would break things over  at https://github.com/gitcoinco/web/blob/master/app/grants/sync/helpers.py#L62

which is an alternate API we use for non-Ethereum Chains

Reply:
Gotcha I'll update that as well